includes/Modules/Search_Console.php

```diff
@@ -303,7 +303,7 @@ protected function parse_data_response( Data_Request $data, $response ) {
 				/* @var Google_Service_SearchConsole_SitesListResponse $response Response object. */
 				$entries     = Sort::case_insensitive_list_sort(
 					$this->map_sites( (array) $response->getSiteEntry() ),
-					'name'
+					'siteURL' // Must match the mapped value.
```

Comment:
See the mapping here https://github.com/google/site-kit-wp/blob/38c5aff94abc4218469b3207d986f9742523b65c/includes/Modules/Search_Console.php#L360-L365